sandbox/main.lua

```diff
@@ -911,12 +915,16 @@ function setConstants()
 	addPlayerShip("Wesson",		"Chavez",		createPlayerShipWesson		,"J")
 	addPlayerShip("Wiggy",		"Gull",			createPlayerShipWiggy		,"J")
 	addPlayerShip("Yorik",		"Rook",			createPlayerShipYorik		,"J")
+	addPlayerShip("Szpieg",		"Ktlitan Breaker",	createPlayerShipSzpieg	,"W")
+	addPlayerShip("Sztylet",	"Ktlitan Feeder",	createPlayerShipSztylet	,"W")
 	makePlayerShipActive("Spyder")		--J
 	makePlayerShipActive("Argonaut")	--J
 	makePlayerShipActive("Nimbus")		--B
 	makePlayerShipActive("Sting")		--W
 	makePlayerShipActive("Sparrow")		--W
 	makePlayerShipActive("Narsil")		--W
+	makePlayerShipActive("Szpieg")		--W
```

Comment:
If this number of player ships in the active list makes the buttons appear with a scroll bar, please reduce the number of active player ships